examples/braket_features/Error_Mitigation_on_Amazon_Braket.ipynb

```diff
@@ -1,509 +1,433 @@
 {
```

Comment:
<p>Is the print necessary?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/286/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>i'll leave the print in for now, so I can remember which task it is :P </p>